doc/user/content/concepts/views.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Views (Non-Materialized and Materialized)
+description: Conceptual page on views.
+menu:
+  main:
+    parent: concepts
+    weight: 15
+    identifier: 'concepts-views'
+---
+
+## Overview
+
+Views represent queries that are saved under a name for reference. Views provide
+a shorthand for the underlying query.
+
+Materialize offers the following types of views:
+
+Type |
+-----|------------
+[**Non-materialized views**](#non-materialized-views) | Results are **not** persisted in durable storage. Non-materialized views can be indexed to maintain and incrementally update results in memory.
+[**Materialized views**](#materialized-views) | Results **are** persisted and incrementally updated in durable storage. Materialized views can be indexed to load the updated results from durable storage into memory.
+
+All views in Materialize are built by reading data from
+[sources](/concepts/sources) and other views.
+
+## Non-materialized views
+
+A non-materialized view saves a query under a name to provide a shorthand for
+referencing the query. The query is **not** executed during non-materialized
+view creation, and a non-materialized view does **not** persist its results in durable storage.
+
+**However**, you can [index](/concepts/indexes/) a non-materialized view to
+**maintain and incrementally update view results** in memory within the cluster.
+This enables queries within the [cluster](/concepts/clusters/) to use the index
+to access view results from memory.  See [Indexes and non-materialized
+views](#indexes-and-non-materialized-views) for more information.
+
+See also:
+
+- [`CREATE VIEW`](/sql/create-view)
+
+### Indexes and non-materialized views
+
+Indexes can improve query performance. In Materialize, non-materialized views
+can be [indexed](/concepts/indexes/). Indexing a non-materialize view causes
+view results to be **maintained and incrementally updated in memory** within the
+[cluster](/concepts/clusters/). The in-memory up-to-date results are accessible
+to queries within the cluster, even for queries that do not use the index
+key(s). Queries from other clusters cannot use the index.
```

Comment:
Should I mention that they can create the indexes in other clusters as appropriate?